templates/graphql-js-starter/README.md

```diff
@@ -0,0 +1,37 @@
+## graphql-server-ts
+
+Starter template using graphback and graphql-js
+
+### Usage
+The project has been created using `graphback`. Run the project using the following steps. 
+- Start the database
+```
+docker-compose up -d
+```
+- Generate resources(schema and resolvers) and create database
+```
+graphback generate
+graphback db
+```
+- Start the server
+```
+npm start
+```
+
+### Directory structure
+
+```
+|-----generated
```

Comment:
I think we might move this from templates to docs.and remove it here. it is hard to maintain it

Reply:
The folder structure ?